spot-controller/src/remote-control.js

```diff
@@ -46,6 +51,11 @@ export default class RemoteControl extends React.PureComponent {
         this._setRef = this._setRef.bind(this);
 
         this._resolveUrl();
+
+        this.userAgentString
+            = __DEV__
+                ? 'SpotController/development'
+                : `${AppInfo.name}/${AppInfo.version}(${AppInfo.buildNumber})`;
```

Comment:
To make sure the blame is put on me, I requested that this user agent still take into account feature version somehow. The reason for having user agent in the first place was to provide web a way to know what features the react native app supports, in case some backwards compatibility (ie disabling features) is needed.